draft-ietf-tls-tls13.md

```diff
@@ -2942,22 +2943,47 @@ Any records following a 1-RTT Finished message MUST be encrypted under the
 application traffic key. In particular, this includes any alerts sent by the
 server in response to client Certificate and CertificateVerify messages.
 
-## Post-Handshake Messages
+## Post-Handshake Messages {#post-handshake}
 
 TLS also allows other messages to be sent after the main handshake.
 These messages use a handshake content type and are encrypted under the application
 traffic key.
 
+The post_handshake TLS extension advertises support for post-handshake message types.
+
+~~~~
+    enum {
+        new_session_ticket(0),
+        key_update(1),
+        client_auth_request(2),
+        (255)
+    } MessageType;
+
+    struct {
+        MessageType message_types<0..2^8-1>;
```

Comment:
If it's going to be "which includes at least one MessageType", this should be 1..2^8-1, not 0...